File: Iniciante/1145.js
var lines = input.split(' ').map(Number); 

let x = lines.shift();
let y = lines.shift();

let cnt = 0;

for (let i = 1; i <= y; i++){
  cnt++;
  if (cnt == x) {
    console.log(i);
    cnt = 0;
  } else {
    process.stdout.write(i + " ");
  }
}

File: Iniciante/1145.py
lines = list(map(int, input().split()))

x = lines.pop(0)
y = lines.pop(0)

cnt = 0

for i in range(1, y+1):
    cnt += 1
    if cnt == x:
        print(i)
        cnt = 0
    else:
        print(i, end=' ')
